packages/manager/modules/support/src/tickets/tickets.controller.js

```diff
@@ -0,0 +1,24 @@
+export default class SupportController {
+  /* @ngInject */
+  constructor(
+    $rootScope,
+    $state,
+    OtrsPopupService,
+  ) {
+    this.$rootScope = $rootScope;
+    this.$state = $state;
+    this.OtrsPopupService = OtrsPopupService;
+  }
+
+  $onInit() {
+    this.$rootScope.$on('ticket.otrs.reload', this.reload.bind(this, { cleanCache: true }));
```

Comment:
Why is this needed ?

Reply:
It has been implemented as is. We plan to removed it for the second part of the refactor.

This part can be then omit (or we can add a comment at least if you prefer).